pom.xml

```diff
@@ -589,6 +589,52 @@ Import-Package: \\
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>com.github.eirslett</groupId>
+          <artifactId>frontend-maven-plugin</artifactId>
+          <version>1.15.0</version>
+          <configuration>
+            <installDirectory>${project.build.directory}</installDirectory>
+          </configuration>
+          <executions>
+            <!-- Install Node.js and npm -->
+            <execution>
+              <id>install-node-and-npm</id>
+              <goals>
+                <goal>install-node-and-npm</goal>
+              </goals>
+              <configuration>
+                <nodeVersion>v20.11.0</nodeVersion>
+                <npmVersion>10.2.5</npmVersion>
+              </configuration>
+            </execution>
+
+            <!-- Install markdownlint-cli -->
+            <execution>
+              <id>npm-install-markdownlint</id>
+              <goals>
+                <goal>npm</goal>
+              </goals>
+              <configuration>
+                <workingDirectory>${project.build.directory}</workingDirectory>
+                <arguments>install markdownlint-cli --no-save</arguments>
```

Comment:
Does it use a fixed version or the latest release? We probably want to pin it so newer versions don't suddenly cause build failures.